docs/modules/maintain-cluster/pages/monitoring.adoc

```diff
@@ -147,6 +147,36 @@ hazelcast:
 ----
 ====
 
+=== Managing Data Access
+
+Management Center allows you to access contents of Hazelcast data structures (for instance map entries) via SQL Browser or Map Browser.
+Management Center can't access the data if at least one member has the data access disabled.
```

Comment:
Does it make sense if we give some brief use cases which may require disabling the data access?